contracts/bridges/optimism/OvmL2CrossDomainMessengerInterface.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+interface OvmL2CrossDomainMessengerInterface {
```

Comment:
Should we prefix the interface with `I`? That's what we did with the Polygon `IFxMessageProcessor` interface.

Reply:
sure, that works too

Reply:
updated